util/cron/nightlysubs.pm

```diff
@@ -117,7 +117,11 @@ sub endMailHeader {
 }
 
 sub endMailChplenv {
-    my $chplenv = `$chplhomedir/util/printchplenv --debug`;
+    my $ch = $chplhomedir;
```

Comment:
I don't think CHPL_HOME should be used in the util/ scripts unless they are specifically trying to access something in the compiler/runtime. Relative paths allow the util/ scripts to be used from one place, while the compiler/runtime/etc are in another, and the test/ dir is in a third place.

The performance graphs don't work today when the compiler is in a different place than the util/ scripts because something relies too heavily on CHPL_HOME (e.g. it expects CHPL_HOME/util/test/perf/ to exist, which isn't true in the tarball).